src/ui/public/saved_objects/saved_objects_client.js

```diff
@@ -126,10 +188,10 @@ export class SavedObjectsClient {
     return this._$http(options)
       .then(resp => get(resp, 'data'))
       .catch(resp => {
-        const respBody = resp.data || {};
+        const respBody = get(resp, 'data', {});
```

Comment:
minor feedback (take it or leave it), but I think it'd be easier to read `_.get` than `get` because there is already a `get` function in this file

Actually, even better if we didn't use lodash at all in favor of native functionality (which I think is what we want to gravitate towards anyway).  This is no different then 
```
const respBody = resp.data || {};
```
right?  for me, that is easier to mentally compute (though probably because I'm not very familliar with lodash).

also the line above could just be `resp.data`.

I guess the benefit of using the lodash `get` version is when you pass it long paths and anywhere along the path something is undefined, it won't crash and burn, but return the default value?  If that's the only benefit, then doesn't seem necessary on a path only one attribute deep (unless it works if resp is also undefined?)